api/src/main/resources/workbench.yaml

```diff
@@ -141,38 +141,61 @@ paths:
         "204":
           description: Account deleted successfully.
 
-  /api/v1/register:
+  /api/v1/id-verification:
```

Comment:
Was there no client code that called the previous "register" endpoint? Is it still possible to create a researcher account locally and get into a valid state? Or is the registration / verification stuff just not yet considered for accessing the frontend?

Reply:
The UI wasn't calling this yet AFAIK. That comes next, with these new endpoints.